appformer-form-modeler/appformer-form-modeler-codegen/appformer-form-modeler-codegen-impl/pom.xml

```diff
@@ -153,6 +153,21 @@
       <scope>test</scope>
       <type>test-jar</type>
     </dependency>
+    <dependency>
+      <groupId>org.kie.workbench.forms</groupId>
+      <artifactId>kie-wb-common-forms-fields</artifactId>
+      <version>7.0.0-SNAPSHOT</version>
```

Comment:
These versions should be specified in `appformer-deps/pom.xml`. Also you should rebase this branch off of master where we are using 7.1.0-SNAPSHOT.